spec/git_spec.rb

```diff
@@ -0,0 +1,36 @@
+require './git_clases.rb'
+
+RSpec.describe Gist do
+    describe "#post" do
+      it "crear gist" do
+        filename = "test.txt"
+        description = "descripción"
+        state = true
+        content = "contenido"
+  
+        gist = described_class.new(filename, description, state, content)
+  
+        uri = URI.parse("https://api.github.com/gists")
+        request = Net::HTTP::Post.new(uri)
+
+        gist.post(uri, request)
+  
+        expect(gist.response_status).to eq 201
+      end
+
+      it "error al crear gist" do
```

Comment:
I'm not sure if these test pass, since they have the exact same steps but one matches with `201` and the other one with `400`. I don't know how that would work 🤔